### DISCUSSION TITLE
How to keep the inline props in Form.Control accepter

### DISCUSSION THREAD

**Discussion by calvin1012** [rsuite/rsuite#3070] [Q&A] [answered]
I have write few code here, simplify say, i want to keep the inline layout props in accepter, or i can only add the Stack inside the accepter function?
https://codesandbox.io/s/wizardly-rui-1dyrm5?file=/src/index.js

**Comment by simonguo** [comment, marked as answer]
Hi @calvin1012 , `layout="inline"` on a Form will only add a `rs-form-inline` className to the Form and will not pass it to the child Form.Control . 
You can define an `inline` property for your `acceptor` component, passed to it via Form.Control, just like you use the `dependence` property.
